pkg/helm/helm_test.go

```diff
@@ -186,15 +186,31 @@ func TestExpandHelmTemplates(t *testing.T) {
 				contentsStr := string(contents)
 
 				for _, value := range values {
-					assert.True(
-						t,
-						strings.Contains(contentsStr, value),
-						"file %s for test case %s does not contain %s: %s",
-						existsSubPath,
-						testCase.description,
-						value,
-						contentsStr,
-					)
+					if strings.Contains(value, "RELEASE-NAME") || strings.Contains(value, "release-name") {
```

Comment:
you can use strings.EqualFold to do a case insensitive comparison

Reply:
Good to know! This is a general enough match that I'd need a `ContainsFold`, which sadly doesn't exist.